package.json

```diff
@@ -217,7 +217,7 @@
     "source-map-support": "^0.5.19",
     "symbol-observable": "^1.2.0",
     "tar": "^6.1.11",
-    "tinygradient": "0.4.3",
+    "tinygradient": "1.1.5",
```

Comment:
With v0 it made sense to have a fixed version but now that they have a non-pre release, we should use a pattern: `^1.1.5`

Reply:
good call